Adamant/Modules/Chat/ViewModel/ChatViewModel.swift

```diff
@@ -938,6 +938,8 @@ final class ChatViewModel: NSObject {
             }
             
             filesPicked = oldFiles
+            inputContentUpdated()
```

Comment:
subscribe to the `filesPicked` property and call `inputContentUpdated()` inside the subscription